xla_native_functions.yaml

```diff
@@ -139,7 +139,7 @@ supported:
   - hardtanh
   - hardtanh_backward
   - index.Tensor
-  - index_add_
+  - index_add
```

Comment:
+ What's the reason we moved to `index_add` instead of `index_add_`?

+ Given the PT dispatcher identifies `index_add_` as the reference (see [this](https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/native_functions.yaml#L5773)), isn't it more appropriate to stay with `index_add_`?

CC @JackCaoG for more context

Reply:
The reference (PT native functions) will be updated once the underpinning PT PR lands; afterwards, it would be more appropriate to manually implement the out-of-place version.